sale_commission_product/models/product_template.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (c) 2015 Asr Oss (http://www.asr-oss.com)
+#                       Alejandro Sanchez Ramirez <alejandro@asr-oss.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
```

Comment:
Yeah, but in that case, the field was a boolean, that it's propagated from product.template to product.product, having the best of both (set at template or at product level). In one2many, you need an inverse many2one field, that can only be of one model, so you need to step down this field to product.product.